*** setup_configure.py	2012-02-29 05:53:48.000000000 +1100
--- /tmp/setup_configure.py	2012-11-07 22:18:13.000000000 +1100
***************
*** 1161,1166 ****
--- 1161,1169 ----
      def __init__( self, setup ):
          UnixCompilerGCC.__init__( self, setup )
  
+     def get_lib_name_for_platform( self, libname ):
+         return '%s.dylib' % (libname,)
+ 
      def _getLdLibs( self ):
          py_ld_libs = [
                  '-L%(SVN_LIB)s',
***************
*** 1178,1183 ****
--- 1181,1187 ----
                  '-lsvn_subr-1',
                  '-lsvn_diff-1',
                  '-lapr-1',
+                 '-lpython'
                  ] )
          return py_ld_libs
  
